File: testdata/remove/expected/file.ts
function greet(name: string): string {
    return `Hello, ${name}!`;
}

class Greeter {
    greeting: string;

    constructor(message: string) {
        this.greeting = message;
    }

    greet(): string {
        return `Hello, ${this.greeting}!`;
    }
}

console.log(greet("World"));

File: testdata/remove/expected/file.py
def say_hello(name="World"):
    """Print a hello message"""
    return f"Hello, {name}!"

class Greeter:
    def __init__(self, greeting="Hello"):
        self.greeting = greeting

    def greet(self, name):
        return f"{self.greeting}, {name}!"

if __name__ == "__main__":
    print(say_hello())